libindy/src/api/did.rs

```diff
@@ -47,29 +51,39 @@ use std::ptr;
 #[no_mangle]
 pub  extern fn indy_create_and_store_my_did(command_handle: i32,
                                             wallet_handle: i32,
-                                            did_json: *const c_char,
+                                            did_info_json: *const c_char,
                                             cb: Option<extern fn(xcommand_handle: i32,
                                                                  err: ErrorCode,
                                                                  did: *const c_char,
                                                                  verkey: *const c_char)>) -> ErrorCode {
-    trace!("indy_create_and_store_my_did: >>> wallet_handle: {:?}, did_json: {:?}", wallet_handle, did_json);
+    trace!("indy_create_and_store_my_did: >>> wallet_handle: {:?}, did_json: {:?}", wallet_handle, did_info_json);
 
-    check_useful_c_str!(did_json, ErrorCode::CommonInvalidParam3);
+    check_useful_c_str!(did_info_json, ErrorCode::CommonInvalidParam3);
```

Comment:
I suggest to define new macro check_useful_json! that will deserialize json to the variable with the same name. It will allow to remove a lot of boilerplate. Usage will be the following:

```
check_useful_json!(did_info, ErrorCode::CommonInvalidParam3, MyDidInfo);
``

Reply:
I wanted to change the code as little as possible. Yes, I'll give it a go.